docs/docs/guides/dashboard/on-premise/install.md

```diff
@@ -131,6 +132,20 @@ The environment variables required to authenticate against S3-compliant storages
 ### Google Cloud Storage
 For Google Cloud Storage, follow [these docs](https://cloud.google.com/storage/docs/authentication/managing-hmackeys) to get the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` pair. The `AWS_ENDPOINT` should be set to `https://storage.googleapis.com`. Other environment variables are the same as for any other S3-compliant storage.
 
+### Version control system configuration
+
+We integrate with providers of your version control system to provide extra features such as posting automatically posting comments in your pull requests.
+
+#### GitHub
+
+You will need to create a GitHub app. You can reuse the one you created for authentication. In the `Permissions and events`'s `Repository permissions` section, you will need to additionally set the `Pull requests` permission to `Read and write`.
```

Comment:
I think we should document this feature too. Otherwise anyone coming across this piece won't know which features they are enabling by setting this up.

Reply:
I will do that in a follow-up once all the pieces of the GitHub app are there.